File: shared/src/commonMain/kotlin/code/yousef/model/Service.kt
package code.yousef.model

data class Service(
    val id: String,
    val title: String,
    val shortDescription: String,
    val fullDescription: String,
    val iconClass: String = "",
    val price: Double? = null,
    val features: List<String> = emptyList(),
    val ctaText: String = "Hire Me",
    val ctaLink: String = "/contact",
    val displayOrder: Int = 0,
    val featured: Boolean = false,
    val createdAt: String = "",
    val updatedAt: String = "",
    val detailsLink: String = ""
) 

File: src/main/resources/web/src/models/Service.ts
export interface Service {
  id: string;
  title: string;
  shortDescription: string;
  fullDescription: string;
  iconClass: string;
  price: number | null;
  features: string[];
  ctaText: string;
  ctaLink: string;
  displayOrder: number;
  featured: boolean;
  createdAt: string;
  updatedAt: string;
  detailsLink: string;
} 